mono/metadata/w32subset.h

```diff
@@ -106,3 +112,39 @@
 #define HAVE_API_SUPPORT_WIN32_WAIT_FOR_INPUT_IDLE 1
 #endif
 #endif
+
+#ifndef HAVE_API_SUPPORT_WIN32_SIGNAL_OBJECT_AND_WAIT
```

Comment:
The intent and prior pattern is 0 or 1. That way another file can define the symbols, not just omit them. Omission is ambiguous. 0 is not.